packages/eas-json/schema/eas.schema.json

```diff
@@ -168,6 +157,16 @@
           "description": "The cache configuration.",
           "markdownDescription": "Cache configuration. This feature is intended for caching values that require a lot of computation, e.g. compilation results (both final binaries and any intermediate files), but it wouldn't work well for `node_modules` because the cache is not local to the machine, so the download speed is similar to downloading from the npm registry."
         },
+        "autoIncrement": {
+          "enum": [false, true],
+          "description": "Controls how EAS CLI bumps your application build version.",
+          "markdownDescription": "Controls how EAS CLI bumps your application build version.\n\nAllowed values:\n- `\"true\"` - for iOS, bumps the last component of `expo.ios.buildNumber` (e.g. `1.2.3.39` -> `1.2.3.40`) and for Android, bumps `expo.android.versionCode` (e.g. `3` -> `4`).\n- `false` - versions won't be bumped automatically (default)\n\nIn the case of a bare project, it also updates versions in native code. For Android, `expo.version` corresponds to `versionName` and `expo.android.versionCode` to `versionCode` in the `build.gradle`. Google Play uses these values to identify the app build, `versionName` is the version visible to users, whereas `versionCode` defines the version number. The combination of those needs to be unique, so you can bump either of them.\nFor iOS, `expo.version` corresponds to `CFBundleShortVersionString` and `expo.ios.buildNumber` to `CFBundleVersion` in the `Info.plist`. The App Store is using those values to identify the app build, `CFBundleShortVersionString` is the version visible to users, whereas `CFBundleVersion` defines the build number. The combination of those needs to be unique, so you can bump either of them.",
```

Comment:
```
Allowed values:
  - true
  - false
```
a list like that looks weird for just a boolean value, maybe sth like

 "When enabled, for iOS, bumps the last component of `expo.ios.buildNumber` (e.g. `1.2.3.39` -> `1.2.3.40`) and for Android, bumps `expo.android.versionCode` (e.g. `3` -> `4`)."
 cc @dsokal

Reply:
Sounds good